docs/source/conf.py

```diff
@@ -48,8 +48,9 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 FILE_PATH = os.path.dirname(__file__)
-VERSION_MODULE = os.path.abspath(os.path.join(FILE_PATH,
-    '../urwid/version.py'))
+VERSION_MODULE = os.path.abspath(
+    os.path.join(FILE_PATH, '../../urwid/version.py')
```

Comment:
Path changed due to hierarchy change

Reply:
this file doesn't exist anymore, does it?

Reply:
it's generated during package build/install. It should be never added to git.